classes.py

```diff
@@ -5,15 +5,59 @@
 
```

Comment:
The Yahtzee scoring card is really nice, I would maybe add it in the readme file, then people can immediately understand much better what the game is actually. One way you can add an image in the readme file is by creating an issue (upload a picture there) and then copy the link of the issue and add it to the readme file like this:
![Name of your file](https://user-images.githubusercontent.com/64072862/thelinkyougeneratedinissues.png)
PS: I just saw the “dice_instruction” pic and this would also be nice in the readme file 😊

Just so you know, your instructions aren’t displayed correctly (at least on my computer the lines are too long. Maybe change the file into markdown format, in that case it automatically goes to a new line. 

I really like this MoSCoW method you used. I might use it also someday!